00-open-a-shapefile.Rmd

```diff
@@ -72,16 +72,15 @@ Crop and extract raster values in R</a>
 NOTE: The data used in this tutorial were collected at Harvard Forest which is
 a the National Ecological Observatory Network field site 
 <a href="http://www.neoninc.org/science-design/field-sites/harvard-forest" target="_blank">
-More about the NEON Harvard Forest field site</a>. These data are proxy data for what will be
-available for 30 years from the NEON flux tower 
+More about the NEON Harvard Forest field site</a>. These data are a preview of the 30 year time-series that will be collected from the NEON flux tower and made available
 [from the NEON data portal](http://data.neoninc.org/ "NEON data").
 {: .notice}
  
 ###Notes about R Libraries
 
-To work with vector data in R, we can use the `rgdal` library. Notice that `sp`
-also automatically loads when `rgdal` is loaded. We will load the `raster`
-library to work with rasters. The `raster` library also allows us to explore
+We will use the `rgdal` library (package) to work with vector data in R. Notice that `sp`
+is automatically loaded as a side-effect when `rgdal` is loaded. We will use the `raster`
```

Comment:
this is great, @jsta why don't we just remove "as a side-effect". `sp` is automatically loaded when `rgdal` is loaded.